crates/harp/src/environment_iter.rs

```diff
@@ -38,14 +36,41 @@ pub enum BindingValue {
         object: RObject,
         data1: RObject,
         data2: RObject,
-        has_nested_environment: BindingNestedEnvironment,
     },
     Standard {
         object: RObject,
-        has_nested_environment: BindingNestedEnvironment,
     },
 }
 
+// Two binding values are equal if they are identical (ie, their SEXP's)
```

Comment:
This is a bit misleading because this notion of equality is very different to R's notion of "identical"